### PR TITLE
Fix polynomial decay with cycle for global step=0

### PR DESCRIPTION
**Pull request by jemaw**
For polynomial decay with cycle=True the learning rate at
step 0 becomes NaN, because in the process of calculating it we
devide by 0. This change should fix it, by setting the multiplier
for the decay steps to one for global_step=0.

### CODE REVIEW

**Comment by caisq** [comment]
@tensorflow-jenkins test this please

**Review by vrv — APPROVED** [review]

**Comment by caisq** [comment]
@tensorflow-jenkins test this please

Checking a potential flake in timeseries:input_pipeline_test
